src/Proj4.jl

```diff
@@ -30,12 +27,7 @@ end
 const version = _version()
 
 # Detect underlying libproj support for geodesic calculations
-const has_geodesic_support = version >= v"4.9.0"
-
-if has_geodesic_support
-    export geod_direct, geod_inverse, geod_destination, geod_distance
-    include("proj_geodesic.jl") # low-level C-facing functions (corresponding to src/geodesic.h)
-end
+const has_geodesic_support = true
```

Comment:
Do we still need this?
If this is not going to change the value, we could delete this variable overall and also delete the references in the tests and in proj_functions.jl
If this is a wanted change, I could prepare a pull request.

Reply:
Thanks for having a look!

Correct, we don't need this internally anymore. At the same time, I don't know if people are using this, and it is fairly harmless to keep around for now.

I guess this can go once we move the package over from the old API to the new one, as in https://github.com/JuliaGeo/Proj4.jl/issues/34#issuecomment-573400435.